iceoryx_posh/source/roudi/port_manager.cpp

```diff
@@ -282,9 +282,12 @@ void PortManager::handleInterfaces() noexcept
         // also forward services from service registry
         /// @todo #415 do we still need this? yes but return a copy here to be stored in shared memory via new
         /// StatusPort's
+        /// @todo iox-#27 I guess this was necessary since a service could be offered via ServiceDiscovery;
```

Comment:
Interesting comment ... does the DDS gateway still work? This I would check out before merging this PR. I have no idea if this is tested in the integration tests.

Reply:
Yes, at least the tests are working and the code only processes `CaproMessageSubType::PUBLISHER`